public/scripts/group-chats.js

```diff
@@ -1309,11 +1330,13 @@ function getGroupCharacterBlock(character) {
     const isFav = character.fav || character.fav == 'true';
     template.data('id', character.avatar);
     template.find('.avatar img').attr({ 'src': avatar, 'title': character.avatar });
-    template.find('.ch_name').text(character.name);
-    template.attr('chid', characters.indexOf(character));
+    template.find('.ch_name').text(character.name + (character.queueOrder > 0?'   (#' + character.queueOrder + ')':''));    template.attr('chid', characters.indexOf(character));
```

Comment:
The pound sign could be a part of the CSS style added by with ::before selector instead of a JS-coded value. That would ofc. require it to be a separate element in the name row.

Reply:
css is the spawn of satan, so I had avoided that.  Moved in latest commit.